ie-solver/boundaries/annulus.cpp

```diff
@@ -0,0 +1,107 @@
+// Copyright 2019 John Paul Ryan
```

Comment:
how about another boundary file with two holes, source + sink

Reply:
We'll leave it up to the holder of the boundary object to change the hole locations and reinitialize if they want